script.js

```diff
@@ -0,0 +1,41 @@
+const hamburger = document.querySelector('#menu');
+const disappear = document.querySelector('#disappear');
+const rotateRight = document.querySelector('#rotate-right');
+const rotateLeft = document.querySelector('#rotate-left');
+const menuList = document.querySelector('#menuList');
+const menuButtons = document.querySelectorAll('.menuButton');
```

Comment:
- [ ] Kindly don't forget that it is required when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear, to improve the user experience :+1: